core/services/eth/contracts/FluxAggregator.go

```diff
@@ -3,8 +3,8 @@ package contracts
 import (
 	"math/big"
 
-	"github.com/smartcontractkit/chainlink/core/eth"
-	ethsvc "github.com/smartcontractkit/chainlink/core/services/eth"
+	"github.com/smartcontractkit/chainlink/core/services/eth"
```

Comment:
I still find it weird/surprising that our `eth` package is split across two subdirectories...

It's in both `/eth` and `/services/eth`... should we do something about that?

Reply:
That's... what this PR does.  It's all one package now.

Reply:
That's exactly the goal of this PR.  There's only one package now.